dist/selleckt-legacy.js

```diff
@@ -861,21 +861,38 @@ _.extend(SingleSelleckt.prototype, {
         matchTerm = term.toLowerCase();
 
         return _.reduce(items, function(memo, item){
-            var matchIndex;
+            var matchIndex, matchStart, matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = item.aliases.reduce(function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
```

Comment:
It's probably quite low down on the things-to-worry-about scale yes, it just crossed my mind..

You're right that we don't check this above, but above (L870) we don't check the type of `label` either and just assume it's always a string..

Feel free to treat this as NABD anyway